### PR TITLE
Cleanup Page Title Logic

### PR DESCRIPTION
**Pull request by utku-ozturk**
CGAP Page title logic is clonned/adapted to 4DN.

### CODE REVIEW

File: src/encoded/static/components/static-pages/HomePage.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ import { requestAnimationFrame } from '@hms-dbmi-bgm/shared-portal-components/es
 
 import { navigate } from'./../util';
 import { Announcements, HomePageCarousel } from './components';
+import { pageTitleViews, PageTitleContainer, TitleAndSubtitleUnder, OnlyTitle } from './../PageTitle';
```
Comment (j1z0) [inline src/encoded/static/components/static-pages/HomePage.js]: Codacy Issue found: ['OnlyTitle' is defined but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4798956)

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
- Added 2
           

Complexity increasing per file
==============================
- src/encoded/static/components/PageTitle.js  5
- src/encoded/static/components/browse/SearchView.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4798956&bid=15971369)

File: src/encoded/static/components/navigation/NavigationBar.js [hunk 2]
```diff
@@ -120,8 +120,8 @@ export class NavigationBar extends React.PureComponent {
 
     render() {
         const { testWarning, mobileDropdownOpen, mounted } = this.state;
-        const { href, context, schemas, browseBaseState, isFullscreen } = this.props;
-        const testWarningVisible = !!(testWarning & !isFullscreen); // Hidden on full screen mode.
+        const { href, context, schemas, browseBaseState, isFullscreen, currentAction } = this.props;
```
Comment (j1z0) [inline src/encoded/static/components/navigation/NavigationBar.js]: Codacy Issue found: ['currentAction' is missing in props validation](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4798956)

**Comment by utku-ozturk** [comment]
[Cleanup Page Title Logic](https://trello.com/c/oms7JgtY/59-cleanup-page-title-logic)

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
- Added 1
           

Complexity increasing per file
==============================
- src/encoded/static/components/PageTitle.js  5
- src/encoded/static/components/browse/SearchView.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4798956&bid=15971369)

File: src/encoded/static/components/navigation/NavigationBar.js (hunk 2 above)
Comment (j1z0) [inline src/encoded/static/components/navigation/NavigationBar.js]: Codacy Issue found: ['currentAction' is missing in props validation](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4798956)

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
- Added 1
           

Complexity increasing per file
==============================
- src/encoded/static/components/PageTitle.js  5
- src/encoded/static/components/browse/SearchView.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4798956&bid=15971369)

File: src/encoded/static/components/navigation/NavigationBar.js (hunk 2 above)
Comment (j1z0) [inline src/encoded/static/components/navigation/NavigationBar.js]: Codacy Issue found: ['currentAction' is missing in props validation](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4798956)

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
- Added 2
           

Complexity increasing per file
==============================
- src/encoded/static/components/PageTitle.js  5
- src/encoded/static/components/browse/SearchView.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4798956&bid=15971369)

File: src/encoded/static/components/navigation/NavigationBar.js (hunk 2 above)
Comment (j1z0) [inline src/encoded/static/components/navigation/NavigationBar.js]: Codacy Issue found: ['currentAction' is missing in props validation](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4798956)

File: src/encoded/static/components/item-pages/UserView.js [hunk 3]
```diff
@@ -767,3 +768,26 @@ export class ImpersonateUserForm extends React.PureComponent {
     }
 
 }
+const UserPageTitle = React.memo(function UserPageTitle(props) {
+    const { alerts, context, session, href, schemas } = props;
```
Comment (j1z0) [inline src/encoded/static/components/item-pages/UserView.js]: Codacy Issue found: ['schemas' is assigned a value but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4798956)

**Pull request merged**